bpfd/src/command.rs

```diff
@@ -601,37 +545,81 @@ impl Program {
 
     pub(crate) fn if_index(&self) -> Option<u32> {
         match self {
-            Program::Xdp(p) => Some(p.info.if_index),
+            Program::Xdp(p) => p.if_index,
```

Comment:
We should probably just use `_ = > None` to avoid having to list out every enum variant that doesn't use an if_index here.